netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[redirects]]
+  from = "/*"
+  to = "https://openzeppelin.com"
```

Comment:
Do we want to add redirects for the old pages in the site, or at least for `security-audits`?

Reply:
I'd also try to keep the redirects for the `jobs` category, since they may have been shared in searches already (both `/jobs` page and individual searches like `/jobs/software-engineer/`). If they carry the same name in the new site, we could use the [`:splat` redirect capture](https://www.netlify.com/docs/redirects/#splats).

Reply:
:splat redirects coming in the next branch.